ui/app/pages/send/send.scss

```diff
@@ -231,9 +230,8 @@
         &__input {
           @extend %col-nowrap;
 
-          font-size: 0.75rem;
-          line-height: 0.75rem;
-          font-weight: 400;
+          @include H7;
```

Comment:
This does increase the height of the ENS input in the send flow (by about 4 pixels), but 🤷‍♂️ It looks OK to me.